docs/av-stack-diff.1.md

```diff
@@ -0,0 +1,17 @@
+# av-stack-diff 1 "" av-cli "Aviator CLI User Manual"
+
+# NAME
+
+av-stack-diff - Show the diff between working tree and parent branch. 
+
+# SYNOPSIS
+
+`av stack diff [-h | --help]`
```

Comment:
We haven't been including `--help` in the other commands I don't think, so let's not include it here for consistency. You can just delete the `OPTIONS` section altogether since there are no other flags.

Reply:
done